packages/block-library/src/query-pagination-next/index.php

```diff
@@ -26,7 +27,13 @@ function render_block_core_query_pagination_next( $attributes, $content, $block
 	if ( $pagination_arrow ) {
 		$label .= $pagination_arrow;
 	}
-	$content = '';
+
+	$wrapper_attributes = get_block_wrapper_attributes();
+	$content            = sprintf(
```

Comment:
Should we escape the output here incase the user tries to insert something untrustworthy in the attribute? (https://developer.wordpress.org/themes/theme-security/data-sanitization-escaping/)

Reply:
Given that its not escaped elsewhere in this file it's probably safe...

Reply:
Yep, I think so